### ISSUE TITLE
FR: Support for space-after-named-function

### ISSUE THREAD

**Issue by rclai** [beautify-web/js-beautify#707]
This should work just like `space-after-anon-function`, but for named function expressions:

``` js
var myFunction = function someName () {};
                                  ^
```

**Comment by bitwiseman** [comment]
Do you mean:
```
var myFunction = function  someName () {};
```

**Comment by rclai** [comment]
Oh yeah, whoops, edited.

**Comment by laggingreflex** [comment]
Related/dupe: #608

**Comment by kanedauy** [comment]
Yes, I want this, I try to use standard and is bothering me with this all the time.

**Comment by mesqueeb** [comment]
I am also trying to recreate the Standard ruleset: https://github.com/beautify-web/js-beautify/issues/1356
Any chance you guys can share your config file how you've recreated the Standard ruleset?

**Comment by MacKLess** [comment]
Closed as a duplicate of #608
